tools/libcloud/mtps.py

```diff
@@ -184,16 +213,35 @@ def wait_for_public_ip(node):
     logging.info("Test will run at: %s", pretty_time(os.environ["RADIX_MTPS_NETWORK_START_PUMP"]))
 
     os.environ["RADIX_MTPS_NETWORK_SEEDS"] = boot_node.public_ips[0]
-    rendered_file = gcp.render_template(config.CORE_CLOUD_INIT)
-    gcp.create_core_template(gce, config.CORE_MACHINE_INSTANCE_TEMPLATE_NAME, open(rendered_file, "r"))
 
-    for region, size in config.CORE_REGIONS.items():
+    # create core node machine template
+    rendered_file = gcp.render_template(config.STORAGE["CORE_CLOUD_INIT"])
+    gcp.create_core_template(gce, config.STORAGE["CORE_MACHINE_INSTANCE_TEMPLATE_NAME"], open(rendered_file, "r"))
+
+    for region, size in config.STORAGE["CORE_REGIONS"].items():
         region = gce.ex_get_region(region)
         count = gcp.count_core_nodes(gce, region)
         if count == 0:
             logging.info("Creating %d Core node in %s...", size, region.name)
-            gcp.create_core_group(gce, region, size, config.CORE_MACHINE_INSTANCE_TEMPLATE_NAME, prefix="cores")
+            gcp.create_core_group(gce, region, size, config.STORAGE["CORE_MACHINE_INSTANCE_TEMPLATE_NAME"])
         elif count < size:
-            logging.warn("Not enough Core nodes in %s than requested: %d < %d", region.name, count, size)
+            logging.warning("Not enough Core nodes in %s than requested: %d < %d", region.name, count, size)
         else:
             logging.info("Core nodes in %s are already up and running", region.name)
+
+    # create extra nodes
+    if "EXTRA_REGIONS" in config.STORAGE:
```

Comment:
I don't see us actually using this functionality rather than composing a whole new `CORE_REGIONS`

Reply:
It's just the continuation of the concept described above. It will be used only for our tests, but I wouldn't like to keep separate branch just for testing purposes.